database/schema/03-staking.sql

```diff
@@ -70,6 +70,7 @@ CREATE TABLE validator_status
     validator_address TEXT    NOT NULL REFERENCES validator (consensus_address) PRIMARY KEY,
     status            INT     NOT NULL,
     jailed            BOOLEAN NOT NULL,
+    tombstoned        BOOLEAN NOT NULL,
```

Comment:
Probably we need to add the `DEFAULT FALSE` notation here since it's going to be a new field inside an already existing table and there might be some problems if we don't do that